release-notes/swan-lake-beta1-release-note.md

```diff
@@ -41,10 +41,33 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 ### Language Updates
 
-#### New Features
-
 #### Improvements
 
+- Worker deadlock detection has been improved to include the `wait` action.
+- `self` of an isolated object no longer needs to be accessed within a `lock` statement unless it is used to access a field that is either not `final` or is not a subtype of `readonly` or `isolated object`.
```

Comment:
```suggestion
- The`self` of an isolated object no longer needs to be accessed within a `lock` statement unless it is used to access a field that is either not `final` or is not a subtype of a `readonly` or `isolated object`.
```

Reply:
We have to say "The `self` variable" if we are using "The".

Why do we need the "a" before "`readonly` or `isolated object`". It's two different types here, like saying "not a subtype of `T1` or `T2`."

```suggestion
- The `self` variable of an isolated object no longer needs to be accessed within a `lock` statement unless it is used to access a field that is either not `final` or is of a type that is not a subtype of `readonly` or `isolated object`.
```